projects/components/src/filtering/filter/parser/types/in-not-in-filter-parser.ts

```diff
@@ -5,13 +5,18 @@ import { FilterOperator } from '../../filter-operators';
 import { SplitFilter } from '../parsed-filter';
 import { AbstractFilterParser } from './abstract-filter-parser';
 
-export class InFilterParser extends AbstractFilterParser<PossibleValuesTypes> {
+export class InNotInFilterParser extends AbstractFilterParser<PossibleValuesTypes> {
   public supportedAttributeTypes(): FilterAttributeType[] {
-    return [FilterAttributeType.String, FilterAttributeType.Number, FilterAttributeType.StringMap];
+    return [
+      FilterAttributeType.String,
+      FilterAttributeType.Number,
+      FilterAttributeType.StringMap,
+      FilterAttributeType.StringArray
```

Comment:
Just curious, why do we need to add `FilterAttributeType.StringArray` here ?

Reply:
good catch - I almost commented on this myself since it's independent of the rest of the change :) What it's doing is adding support both for the existing IN operator and the new NOT_IN operator to support string array attributes. For example, MyEntity.labels IN [label1, label2]. So an array on both sides of the operator.